workers/social/lib/social/main.coffee

```diff
@@ -138,7 +138,7 @@ do ->
 
   options = { rateLimitOptions : KONFIG.nodejsRateLimiter }
 
-  app.post '/remote.api/:model/:id?', (require './remoteapi') koding
+  app.post '/remote.api/:token?/:model/:id?', (require './remoteapi') koding
```

Comment:
in some places at our logging infra, we are outputting the urls, having token in the uri might create problems by leaking them. having them in the header would be safer but harder to use.